Library/Homebrew/cmd/desc.rb

```diff
@@ -0,0 +1,59 @@
+require 'descriptions'
+require 'cmd/search'
+
+USAGE =<<EOM
```

Comment:
Since none of internal commands has a usage message. I think we should drop this. Manpage should be sufficiency.